src/contracts/AaveStewardInjectorCollateral.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import {IRiskOracle} from './dependencies/IRiskOracle.sol';
+import {IRiskSteward} from '../interfaces/IRiskSteward.sol';
+import {AaveStewardInjectorBase} from './AaveStewardInjectorBase.sol';
+import {IAaveV3ConfigEngine as IEngine} from 'aave-v3-origin/src/contracts/extensions/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {EngineFlags} from 'aave-v3-origin/src/contracts/extensions/v3-config-engine/EngineFlags.sol';
+import {Strings} from 'openzeppelin-contracts/contracts/utils/Strings.sol';
+import {IAToken} from 'aave-v3-origin/src/contracts/interfaces/IAToken.sol';
+
+/**
+ * @title AaveStewardInjectorCollateral
+ * @author BGD Labs
+ * @notice Aave chainlink automation-keeper-compatible contract to perform collateral update injection
+ *         on risk steward using the edge risk oracle.
+ */
+contract AaveStewardInjectorCollateral is AaveStewardInjectorBase {
+  using Strings for string;
+
+  /**
+   * @param riskOracle address of the edge risk oracle contract.
+   * @param riskSteward address of the risk steward contract.
+   * @param owner address of the owner of the stewards injector.
+   * @param guardian address of the guardian of the stewards injector.
+   */
+  constructor(
+    address riskOracle,
+    address riskSteward,
+    address owner,
+    address guardian
+  ) AaveStewardInjectorBase(riskOracle, riskSteward, owner, guardian) {}
+
+  /// @inheritdoc AaveStewardInjectorBase
+  function getUpdateTypes() public pure override returns (string[] memory updateTypes) {
+    updateTypes = new string[](3);
+    updateTypes[0] = 'ltv';
+    updateTypes[1] = 'liquidationThreshold';
+    updateTypes[2] = 'liquidationBonus';
+  }
+
+  /// @inheritdoc AaveStewardInjectorBase
+  function _injectUpdate(
+    IRiskOracle.RiskParameterUpdate memory riskParams
+  ) internal override {
+    address underlyingAddress = IAToken(riskParams.market).UNDERLYING_ASSET_ADDRESS();
+    uint256 collateralValue = uint256(bytes32(riskParams.newValue));
```

Comment:
treated differently to [this ](https://github.com/bgd-labs/aave-v3-risk-stewards/blob/09f298307fd69dc67314ee4e2873d63dc6d91412/src/contracts/AaveStewardInjectorCaps.sol#L47-L49)